docs/application/dotnet/guides/connectivity/bluetooth.md

```diff
@@ -1078,22 +1078,14 @@ To discover nearby LE devices, perform an LE scan operation:
             }
         }
 
-        IEnumerable<BluetoothLeServiceData> serviceList = leDevice.GetServiceDataList();
+        IEnumerable<BluetoothLeServiceData> serviceList = leDevice.GetServiceDataList(BluetoothLePacketType.BluetoothLeAdvertisingPacket);
```

Comment:
Is it ok to use GetServiceDataList? I can see this method is also in the deprecated list in http://suprem.sec.samsung.net/jira/browse/TCSACR-238. If I misunderstood something please let me know.

Reply:
Yes. GetServiceDataList(void) was duplicated as you said. And, GetServiceDataList(BluetoothLePacketType packetType) is added on below ACR.
http://suprem.sec.samsung.net/jira/browse/TCSACR-237
Thank you.